app/services/make_an_offer.rb

```diff
@@ -87,15 +114,23 @@ def validate_course_data
     errors.add(:offered_course, "does not belong to provider #{current_provider.code}, it belongs to #{provider.code}") if providers_dont_match
   end
 
-  def validate_offer_conditions
-    return if @offer_conditions.blank?
+  def validate_further_conditions
+    return if further_conditions.blank?
 
-    unless @offer_conditions.is_a?(Array)
-      errors.add(:offer_conditions, 'must be an array')
-      return
+    further_conditions.each_with_index do |value, index|
+      errors.add("further_conditions#{index}", "has a condition over #{MAX_CONDITION_LENGTH} chars in length") if value && value.length > MAX_CONDITION_LENGTH
```

Comment:
Error message should follow the GOV.UK Design System guidelines:

[whatever it is] must be 255 characters or fewer

Example: Third condition must be 255 characters or fewer

Reply:
The errors should relate to the individual field, not the fieldset in this case.

There could be different conditions with different error messages.